src/Routes/Signatures/Details.js

```diff
@@ -29,84 +28,86 @@ const Details = () => {
     const sigMetadata = sigDetailsData?.metadata;
+    const isMatched = sigDetailsData?.affectedHosts?.totalCount > 0;
     const references = sigMetadata?.references || [];
     const xfti_references = sigMetadata?.xfti_references || [];
     const dataNotAvailable = intl.formatMessage(messages.dataNotAvailable);
     const detailBlock = (title, detail) => <React.Fragment>
         <p className='ins-l-detailBlockHeader'>{title}</p>
         <p>{detail || dataNotAvailable}</p>
     </React.Fragment>;
-    const referenceLink = (referenceLinkData) => <p>
-        <a rel="noopener noreferrer" target="_blank" href={referenceLinkData}>{referenceLinkData}</a>
-    </p>;
-    const referenceGridItem = (title, links) => <React.Fragment>
-        {links.length > 1 ?
-            <ExpandableSection toggleText={intl.formatMessage(title)} isIndented>
-                {links.map((link) => referenceLink(link))}
-            </ExpandableSection>
-            :
-            <div className='ins-l-referenceLink'>
-                {detailBlock(intl.formatMessage(title), referenceLink(links))}
-            </div>}
+    const referenceLink = (referenceLink, referenceName) => <React.Fragment>
+        <a className='ins-l-referenceLink' rel="noopener noreferrer" target="_blank" href={referenceLink}>{referenceName} <ExternalLinkAltIcon /></a>
     </React.Fragment>;
 
+    const matchedSystemsCount = () => {
+        return isMatched ? <React.Fragment>
+            <ExclamationCircleIcon color='red'/>&nbsp;
+            <span style={{ color: 'red', fontWeight: 'bold' }}>
+                {`${sigDetailsData?.affectedHosts?.totalCount}/${data?.hosts?.totalCount}`}
+            </span>
+        </React.Fragment>
+            :
+            <span>{`${sigDetailsData?.affectedHosts?.totalCount}/${data?.hosts?.totalCount}`}</span>;
+    };
+
     return <React.Fragment>
         <PageHeader>
             <Breadcrumb items={breadcrumbs} />
             <Grid hasGutter>
                 <GridItem md={12} sm={12}>
-                    <Title headingLevel='h1' size={TitleSizes['3xl']} >{sigId}</Title>
+                    <Title headingLevel='h1' size={TitleSizes['3xl']} >{sigId}&nbsp;
+                        {sigMetadata?.threat_type &&
+                            <Label variant="outline" color="blue">{sigMetadata.threat_type}</Label>
+                        }&nbsp;
+                        {isMatched &&
+                            <Label variant="filled" color="red">Matched</Label>
+                        }
+                    </Title>
                 </GridItem>
 
-                {isCodeEditorEnabled && (<GridItem md={7} sm={12}>
-                    { sigDetailsData && <CodeEditor height='250px' code={sigDetailsData.rawRule} isReadOnly isDownloadEnabled isCopyEnabled />}
-                </GridItem>)}
-
                 {loading ? <Loading /> :
-                    <GridItem md={isCodeEditorEnabled && 5 || 12} sm={12}>
+                    <GridItem md={12} sm={12}>
```

Comment:
GridItem defaults to 12, so you do not have to write this as far as I remember.

Reply:
Done!  Thank you